contracts/RootChain.sol

```diff
@@ -455,35 +482,48 @@ contract RootChain {
         // Separate the inputs transactions.
         RLP.RLPItem[] memory splitInputTxs = _inputTxs.toRLPItem().toList();
 
+        uint256[3] memory vars;
```

Comment:
C'mon, no comment? explain why `vars` is used exactly and move the explanation the binding of `vars[i]`-its meaning to the body of that comment as additional explanation.

Reply:
done